packages/contentstack-utilities/src/helpers.ts

```diff
@@ -8,3 +9,20 @@ export const doesBranchExist = async (stack, branchName) => {
       return error;
     });
 };
+
+export const isManagementTokenValid = async (stackAPIKey, managementToken) => {
+  const httpClient = new HttpClient({ headers: { api_key: stackAPIKey, authorization: managementToken } });
+  try {
+    const response = (await httpClient.get(`${configHandler.get('region').cma}/v3/environments?limit=1`))?.data;
```

Comment:
"${configHandler.get('region').cma" consider a case where region is set and the user went to add the token, can you check the case ? handle the case if it is not handled the case already.